### PR TITLE
error message and filename updated for ShareGetSharedWithMeTest

### PR DESCRIPTION
**Pull request by S-Panta**
Related issue : https://github.com/owncloud/ocis-php-sdk/issues/131
The foldername `ShareTestGetSharedWithMeTest` is changed to `ShareGetSharedWithMeTest` for coherent name as all our file is named in similar style.

### CODE REVIEW

File: tests/integration/Owncloud/OcisPhpSdk/ShareGetSharedWithMeTest.php [hunk 1]
```diff
@@ -74,24 +87,38 @@ public function testGetMultipleShareWithMe(): void
...
+                ShareReceived::class,
+                $receivedShare,
+                "Expected class to be " . ShareReceived::class
+                . " but found " . get_class($receivedShare)
+            );
+        }
+        $this->assertCount(
+            2,
+            $receivedShares,
+            "Expected two shares but found ".count($receivedShares)
```
Comment (phil-davis) [inline tests/integration/Owncloud/OcisPhpSdk/ShareGetSharedWithMeTest.php:102]:
```suggestion
            "Expected two shares but found " . count($receivedShares)
```

File: tests/integration/Owncloud/OcisPhpSdk/ShareGetSharedWithMeTest.php [hunk 2]
```diff
@@ -57,9 +57,22 @@ public function testGetShareWithMe(): void
     {
         $this->folderToShare->invite($this->einstein, $this->editorRole);
         $receivedShare = $this->einsteinOcis->getSharedWithMe();
-        $this->assertInstanceOf(ShareReceived::class, $receivedShare[0]);
-        $this->assertEquals($this->folderToShare->getName(), $receivedShare[0]->getName());
-        $this->assertSame($this->folderToShare->getId(), $receivedShare[0]->getRemoteItemId());
+        $this->assertInstanceOf(
+            ShareReceived::class,
+            $receivedShare[0],
+            "Expected class to be 'ShareReceived' but found "
```
[the remark this answers is not in the document]
Comment (S-Panta) [reply tests/integration/Owncloud/OcisPhpSdk/ShareGetSharedWithMeTest.php:63]: This shouldn't be changed at least now for consistency.

File: tests/integration/Owncloud/OcisPhpSdk/ShareGetSharedWithMeTest.php [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+        );
+        $this->assertStringContainsString(
+            'Admin',
+            $receivedShare->getOwnerName(),
+            "Expected owner name to be 'Admin' but found " . $receivedShare->getOwnerName()
+        );
+        $this->assertGreaterThanOrEqual(
+            1,
+            strlen($receivedShare->getOwnerId()),
+            "Exoected the length of ownerId of receive share to be greater than 1"
```
Comment (amrita-shrestha) [inline tests/integration/Owncloud/OcisPhpSdk/ShareGetSharedWithMeTest.php:140]:
```suggestion
            "Expected the length of ownerId of receive share to be greater than 1"
```

File: tests/integration/Owncloud/OcisPhpSdk/ShareGetSharedWithMeTest.php [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+        $this->assertGreaterThanOrEqual(
+            1,
+            strlen((string)$receivedShare->getId()),
+            " The length of received share id to be greater than 1 "
+        );
+        $this->assertNotNull($receivedShare->getParentDriveId(), "Expected Parent drive id to not be null");
+        $this->assertGreaterThanOrEqual(
+            1,
+            strlen((string)$receivedShare->getParentDriveId()),
+            "Exoected the length of parent drive id to be greater than 1"
```
Comment (amrita-shrestha) [inline tests/integration/Owncloud/OcisPhpSdk/ShareGetSharedWithMeTest.php:87]:
```suggestion
            "Expected the length of parent drive id to be greater than 1"
```

**Review by amrita-shrestha — APPROVED** [review]
lgtm

**Pull request merged**